syntax = "proto3";
import "google/protobuf/empty.proto";
service IllumiReadSwypeScript {
  rpc ContextRPC(ContextRPCRequest) returns (ContextRPCResponse);
  rpc HandSwipe2WordRPC(HandSwipe2WordRPCRequest) returns (google.protobuf.Empty);
  rpc Tap2CharRPC(Tap2CharRPCRequest) returns (Tap2CharRPCResponse);
}
message ContextRPCRequest {
  string input0 = 1;
}
message ContextRPCResponse {
  string message = 1;
}
message HandSwipe2WordRPCRequest {
  float localX = 1;
  float localY = 2;
}
message Tap2CharRPCRequest {
  float input0 = 1;
  float input1 = 2;
}
message Tap2CharRPCResponse {
  string message = 1;
}